src/components/Envelope.vue

```diff
@@ -251,6 +261,7 @@ import IconCreateEvent from 'vue-material-design-icons/Calendar'
 import CheckIcon from 'vue-material-design-icons/Check'
 import ChevronLeft from 'vue-material-design-icons/ChevronLeft'
 import DeleteIcon from 'vue-material-design-icons/Delete'
+import ArchiveIcon from 'vue-material-design-icons/Archive'
```

Comment:
for the archive icon, we use /PackageDown

Reply:
Fixed it